### PR TITLE
[MRCNN] Update 2.0 rules to accommodate small batch sizes

### PR DESCRIPTION
**Pull request by ShriyaRishab**
We noticed that the hyperparameters for mask-rcnn only make sense for batch size 16 and above, given the way the reference works.  To allow submissions with batch size < 16, we propose to allow the learning rate to scale with 0.2 * 1/K as well.

### CODE REVIEW

**Comment by ShriyaRishab** [comment]
recheck

**Comment by johntran-nv** [comment]
@mlcommons/wg-training Can folks please review this ahead of this week's working group meeting?  Sorry for the late change but we just noticed this.  

Tagging some folks explicitly as well: @petermattson @emizan76 (can't find eric's handle)

**Comment by emizan76** [comment]
You mean 0.02 * (1/K), right? Has the convergence behavior for low batch sizes been studied?

**Comment by petermattson** [comment]
OK, hearing no objection, going to let this one in.
